### PR TITLE
Feature/cedar integration

### PR DESCRIPTION
**Pull request by adlius**
-   Ticket: []
-   Feature flag: n/a

## Purpose

Merge `feature/cedar-integration` into `develop`.

### CODE REVIEW

File: lib/osf-components/addon/components/file-metadata-form/styles.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors
+
+.node-type-heading {
+    border-top: 2px solid #ddd;
+    border-top: 2px solid rgb(221, 221, 221);
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/file-metadata-form/styles.scss:5]:
```suggestion
    border-top: 2px solid $color-border-gray;
```

File: app/models/cedar-metadata-record.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * CedarMetadataRecordModel
+ *
+ * @description The cedar metadata record model
+ */
+export default class CedarMetadataRecordModel extends OsfModel {
+    @attr('object') metadata!: any;
+    @attr('boolean') isPublished!: boolean;
+
+    @tracked templateName!: string;
```
Comment (futa-ikeda) [inline app/models/cedar-metadata-record.ts:19]:
```suggestion
```

**Review by futa-ikeda — DISMISSED** [review]
Looks good to me! Leaving some optional suggestions to address some leftovers

File: app/models/cedar-metadata-record.ts (hunk 2 above)
Comment (futa-ikeda) [inline app/models/cedar-metadata-record.ts:19]: Note: Thought this was unused and leftover, but this is used as an alias for the `template.name` in the code.

**Review by brianjgeiger — APPROVED** [review]

**Review by futa-ikeda — APPROVED** [review]

**Pull request merged**